### DISCUSSION TITLE
Now that Node 16 is the Active LTS, when can we expect Vercel to announce support for it?

### DISCUSSION THREAD

**Discussion by osdiab** [vercel/vercel#6891] [Ideas] [answered]
It would simplify the onboarding process for my new engineers, since Node 14 and other dependencies my project uses like Hasura's Docker image doesn't natively support Arm64 architecture - Vercel is the reason I haven't bumped our project to Node 16, so once they support it I can make my team's tooling simpler.

**Comment by leerob** [comment, marked as answer]
https://github.com/vercel/vercel/discussions/7003
